crates/sqlexec/src/planner/physical_plan/create_tunnel_exec.rs

```diff
@@ -0,0 +1,62 @@
+use datafusion::physical_plan::{DisplayAs, ExecutionPlan, Partitioning, Statistics};
+use protogen::metastore::types::options::TunnelOptions;
+
+use crate::planner::EMPTY_SCHEMA;
+
+#[derive(Debug)]
+pub struct CreateTunnelExec {
+    pub name: String,
+    pub if_not_exists: bool,
+    pub options: TunnelOptions,
+}
```

Comment:
it's important that similar to the `CreateTunnel` struct, that we can convert this to/from protos. 

I'd expect the table scans to be a little more difficult, as we may need to create a connection during logical planning, then another connection during physical planning (or create a connection pool). something like that 🤷